src/app/components/user/hidUsers.controller.js

```diff
@@ -0,0 +1,135 @@
+hidControllers.controller('hidUsersCtrl', ['$scope', '$rootScope', '$location', '$routeParams', 'gettextCatalog', 'alertService', 'hrinfoService', 'userService', 'User', 'List',
+  function ($scope, $rootScope, $location, $routeParams, gettextCatalog, alertService, hrinfoService, userService, User, List) {
+      $scope.request = {};
+      $scope.totalItems = 0;
+      $scope.itemsPerPage = 50;
+      $scope.currentPage = 1;
+      $scope.selectedFilters = {};
+      $scope.searchTerm = $routeParams.name;
+      var currentSortOrder = $scope.request.name;
+      var defaultRequest = {
+        limit: $scope.itemsPerPage,
+        offset: 0,
+        sort: 'name'
+      };
+
+      if ($routeParams.list) {
+        defaultRequest['lists.list'] = $routeParams.list;
```

Comment:
I think this will not work, because depending on the type of list a user is checked into, this can be operations.list, organizations.list etc... That's why I need the List to be resolved before calling getUsers, and that's why I need this notify (so that the list type is known and correct request can be made)